wa/workloads/deepbench/__init__.py

```diff
@@ -104,7 +104,7 @@ def setup(self, context):
     def run(self, context):
```

Comment:
The previous commit seems to be included in this PR by accident.

Reply:
Erm it is, but only because I thought it has been committed already. I'll adjust  the PR title/text.